Sources/PointFree/Routers/PointFreeRoutes.swift

```diff
@@ -159,8 +159,13 @@ private let routers: [Router<Route>] = [
   .admin <<< .freeEpisodeEmail <<< .index
     <¢> get %> lit("admin") %> lit("free-episode-email") <% end,
 
-  .admin <<< .newEpisodeEmail <<< .send
-    <¢> post %> lit("admin") %> lit("new-episode-email") %> pathParam(.int >>> .tagged) <% lit("send") <% end,
+  PartialIso.admin <<< PartialIso.newEpisodeEmail <<< PartialIso.send
+    <¢> post %> lit("admin") %> lit("new-episode-email") %> pathParam(.int >>> .tagged)
+    <%> formField("subscriber_announcement", .string).map(Optional.iso.some)
+    <%> formField("nonsubscriber_announcement", .string).map(Optional.iso.some)
+    <%> isTest
+    <% lit("send")
```

Comment:
this route is gross :/

notice that the last `/send` path param goes AFTER the form fields. I couldn't figure out how to parse out that part and discard the `Unit`. maybe we can look at this together today.

Reply:
I thought `end` did that...

Reply:
I was able to figure this out, just wasn't thinking clearly before.